CMakeLists.txt

```diff
@@ -64,6 +64,11 @@ if (WIN32)
     endif()
 endif()
 
+cmake_policy(SET CMP0063 NEW)
```

Comment:
You cannot set that policy unconditionnaly. It was [introduced in CMake 3.3[doc], and that triggered a CMake in [travis]. To be compatible with all versions of CMake, you can write:
```cmake
if(POLICY CMP0063)
  cmake_policy(SET CMP0063 NEW)
endif()
```

[doc]: https://cmake.org/cmake/help/v3.3/policy/CMP0063.html
[travis]: https://travis-ci.org/doxygen/doxygen/jobs/271596022#L771

Reply:
Thanks, addressed with v3. Currently running through the Travis checks.